omp/CMakeLists.txt

```diff
@@ -27,8 +27,13 @@ target_sources(ginkgo_omp
         stop/residual_norm_reduction_kernels.cpp)
 
 ginkgo_compile_features(ginkgo_omp)
+
 target_link_libraries(ginkgo_omp PRIVATE "${OpenMP_CXX_LIBRARIES}")
-target_compile_options(ginkgo_omp PRIVATE "${OpenMP_CXX_FLAGS}")
+target_include_directories(ginkgo_omp PRIVATE "${OpenMP_CXX_INCLUDE_DIRS}")
+# We first separate the arguments, otherwise, the target_compile_options adds it as a string
+# and the compiler is unhappy with the quotation marks.
+separate_arguments(OpenMP_SEP_FLAGS NATIVE_COMMAND "${OpenMP_CXX_FLAGS}")
```

Comment:
Nice solution! And the OpenMP target would not set these flags correctly?

Reply:
It was, but there was a problem, when you would do the test_install step.